# Define the goal

# Specify aruco ID to look for
uint32 aruco_id

# Specify the camera tilt_angle at which to scan
float32 tilt_angle

# If robot should rotate to cover the blind spot generated by the mast
bool fill_in_blindspot_with_second_scan 

# If scan should stop as soon as a matching aruco ID is found
bool fast_scan

---
# Define the result

# Whether goal aruco ID was found
bool aruco_found

---
# Define a feedback message

# Pan angle of the camera
float32 pan_angle